packages/replay-web/src/draw.ts

```diff
@@ -254,16 +250,16 @@ const drawUtils = (ctx: CanvasRenderingContext2D) => ({
       ctx.stroke();
     }
   },
-  text(
-    font: TextureFont,
-    text: string,
-    align: "left" | "center" | "right",
-    fillStyle: string
-  ) {
-    const fontString = `${font.size}px ${font.name}`;
-    ctx.font = fontString;
-    ctx.textBaseline = "middle";
-    ctx.textAlign = align;
+  text(font: TextureFont, text: string, fillStyle: string) {
+    const fontString = `
+      ${font.style || "normal"} 
+      ${font.weight || "normal"} 
+      ${font.size ? `${font.size}px` : ""} 
+      ${font.family ? `"${font.family}"` : ""}
+    `;
+    ctx.font = fontString.replace(/\n/g, " ");
```

Comment:
I appreciate this makes the code easier to read, but since it could be called in render at 60 FPS I'd prefer to avoid this regex replace every time.